tools/BUILD

```diff
@@ -21,3 +22,21 @@ buildifier(
     lint_mode = "warn",
     mode = "fix",
 )
+
+py_binary(
```

Comment:
It's too bad you can't easily add args - if you specify args on the command line, they blow away the defaults. I guess that's fine because all the other parts of this script have good defaults and we can create more binary wrappers with different args for other use cases.

Reply:
> they blow away the defaults

this doesn't seem to be happening for me. for example, this is working:

```
bazel run //tools:dev_cluster -- --nodes 1
```

Reply:
Oh awesome!